Makefile

```diff
@@ -65,8 +65,8 @@ deploy: manifests
 # Generate manifests e.g. CRD, RBAC etc.
 .PHONY: manifests
```

Comment:
Would you mind fixing `make revendor` as well? I see following error:
```
github.com/gardener/etcd-druid/hack imports
        github.com/gardener/gardener/.github: malformed import path "github.com/gardener/gardener/.github": leading dot in path element
github.com/gardener/etcd-druid/hack imports
        github.com/gardener/gardener/hack/.ci: malformed import path "github.com/gardener/gardener/hack/.ci": leading dot in path element
make: *** [Makefile:31: revendor] Error 1
```

Reply:
Which Golang version do you use? I think this was a bug in Go 1.16.{0,1} (see this issue https://github.com/golang/go/issues/43985).

Reply:
I see. Thanks.